indico/modules/events/surveys/models/surveys.py

```diff
@@ -103,6 +103,7 @@ class Survey(db.Model):
         'SurveyQuestion',
         cascade='all, delete-orphan',
         lazy=True,
+        order_by='asc(SurveyQuestion.position)',
```

Comment:
I think `order_by='SurveyQuestion.position'` is already enough; ASC is the default.